prow/plugins/approve/approve_test.go

```diff
@@ -148,6 +150,40 @@ func (fr fakeRepo) IsNoParentOwners(path string) bool {
 	return false
 }
 
+func (fr fakeRepo) ParseSimpleConfig(path string) (repoowners.SimpleConfig, error) {
```

Comment:
Is this logic different for the non-fake repo client? can we unify them?

Reply:
> Is this logic different for the non-fake repo client? 

Do you mean the real OwnersClient? The logic is not different. :+1: 

> can we unify them?

Do you mean adding something like a `FakeOwnersClient` to a `fakerepoowners.go`? That's possible, but I'd prefer to do that as a follow up, if that's ok, because it would mean changing a bunch of things for other plugins where `repoowners` is used.

Reply:
I meant more that anyone using this fake is doing to be dependent on this implementation of the parsing, not the one that runs in production, so while they think they are testing the repo owners interaction they are not